pkg/image/api/types.go

```diff
@@ -50,6 +50,12 @@ type Image struct {
 	DockerImageManifest string
 	// DockerImageLayers represents the layers in the image. May not be set if the image does not define that data.
 	DockerImageLayers []ImageLayer
+	// DockerImageSignatures provides the signatures on this JWS as opaque blobs.
+	DockerImageSignatures [][]byte
```

Comment:
@miminar does this signatures has to do something with that PR of yours, or are these totally separate? If yes - are you going to refactor this afterwards?